frontend/src/metabase/collections/components/CollectionSidebar/Bookmarks/Bookmarks.styled.jsx

```diff
@@ -13,8 +14,32 @@ export const BookmarkTypeIcon = styled(Icon)`
   opacity: 0.5;
 `;
 
-export const BookmarkLinkRoot = styled.div`
+export const BookmarkListRoot = styled.div`
   margin: ${space(2)} 0;
 `;
 
+export const BookmarkContainer = styled.div`
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+
+  &:hover {
+    background: ${color("bg-medium")};
+
+    button {
+      opacity: 0.5;
+    }
+  }
+
+  button {
```

Comment:
WDYT about extracting this button as a separate `@styled` component instead of styling it with a nested selector?